saq/queue/postgres_ddl.py

```diff
@@ -0,0 +1,15 @@
+CREATE_JOBS_TABLE = """
+CREATE TABLE IF NOT EXISTS {jobs_table} (
+    key TEXT PRIMARY KEY,
+    lock_id BIGSERIAL,
+    queued BIGINT NOT NULL DEFAULT EXTRACT(EPOCH FROM now()),
+    job JSONB,
+    queue TEXT,
+    status TEXT,
+    scheduled BIGINT
+);
+"""
+
+CREATE_JOBS_DEQUEUE_INDEX = """
+CREATE INDEX IF NOT EXISTS saq_jobs_dequeue_idx ON {jobs_table} (status, queue, (queued + scheduled))
```

Comment:
does this index work? queued + scheduled? does that get hit?

Reply:
why do we have to do math? is that how it was done? is scheduled not just a timestamp it's an integer seconds?